pkg/deviceshifu/deviceshifuHTTP/deviceshifuHTTP.go

```diff
@@ -1,4 +1,4 @@
-package deviceshifuHTTP
+package deviceshifuhttp
```

Comment:
let's rename the folder itself to `deviceshifuhttp` too, same for other deviceshifu protocol packages and cmds